### PR TITLE
chore: update terminals in README

### PR DESCRIPTION
**Pull request by Beta-Computer**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Doc update 

updated the readme to include [Tabby](https://tabby.sh) as supported terminal

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,7 +125,7 @@ Never submitted a PR before? Check out our [How to Contribute](https://fig.io/do
 
 #### What terminals does Fig work with?
 
-Fig works with iTerm, the native MacOS Terminal app, Hyper and the integrated terminal in VSCode.
+Fig works with iTerm, [Tabby](https://tabby.sh), the native MacOS Terminal app, Hyper and the integrated terminal in VSCode.
```
Comment (fedeci) [inline README.md:128]: Can we add links to the other terminal pages too, please?

Comment (Beta-Computer) [reply README.md]: Yes please see new commit

File: README.md [hunk 2]
```diff
@@ -125,7 +125,7 @@ Never submitted a PR before? Check out our [How to Contribute](https://fig.io/do
 
 #### What terminals does Fig work with?
 
-Fig works with iTerm, the native MacOS Terminal app, Hyper and the integrated terminal in VSCode.
+Fig works with [iTerm](https://iterm2.com/), [Tabby](https://tabby.sh), the native MacOS Terminal app, Hyper and the [integrated terminal in VSCode](https://code.visualstudio.com/docs/editor/integrated-terminal).
```
Comment (SeparateRecords) [inline README.md:128]:
```suggestion
Fig works with the native MacOS Terminal app, [iTerm](https://iterm2.com/), [Tabby](https://tabby.sh), [Hyper](https://hyper.is) and the [integrated terminal in VSCode](https://code.visualstudio.com/docs/editor/integrated-terminal).
```

**Comment by Beta-Computer** [comment]
Could link terminal to [this](https://support.apple.com/en-au/guide/terminal/welcome/mac)

**Comment by fedeci** [comment]
I believe the preinstalled terminal is know by every dev so we don't require a link. 
Thanks for the contribution!

**Pull request merged**